avm/res/db-for-my-sql/flexible-server/tests/e2e/max/dependencies2.bicep

```diff
@@ -31,15 +31,16 @@ resource keyVault 'Microsoft.KeyVault/vaults@2023-02-01' = {
     }
     tenantId: tenant().tenantId
     enablePurgeProtection: true // Required for encryption to work
-    softDeleteRetentionInDays: 7
+    softDeleteRetentionInDays: 90
```

Comment:
Please no 😄 Does the service not allow `7`? 
We try to keep it as low as possible

Reply:
Fixed.